Templates/CDS-CSharp/run.csx

```diff
@@ -0,0 +1,67 @@
+﻿#load ".\SecurityHelper.csx"
+#load ".\TraceWriterTelemetryBridge.csx"
+
+using Microsoft.CommonDataService;
+using Microsoft.CommonDataService.CommonEntitySets;
+using Microsoft.CommonDataService.Configuration;
+using Microsoft.CommonDataService.ServiceClient.Security;
+using Microsoft.CommonDataService.ServiceClient.Security.Credentials;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+class EnvironmentValues
```

Comment:
I believe there are discussions going on about the design for this. But yes, appsettings is one of the probable places. Nimak and Matthew Henderson are the PMs on this.